notebooks/00_core.ipynb

```diff
@@ -29,7 +29,10 @@
     "import wget\n",
```

Comment:
<p>do you need these imports?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>good catch! it's an artifact from some experimenting i did - will fix it</p><p> </p>